.github/workflows/backend.yml

```diff
@@ -130,14 +130,31 @@ jobs:
     name: requirements check
     runs-on: ubuntu-20.04
     timeout-minutes: 3
-    # Assuming Python 3.8 in GHA's ubuntu 20.04. If we run into any issues,
-    # this can just be pinned with setup-python.
     steps:
+      - uses: getsentry/action-github-app-token@v1
+        id: token
+        continue-on-error: true
+        with:
+          app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
+          private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
       - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.8.13
       - name: check requirements
         run: |
-          python -m pip install 'pip-tools==6.7.0'
-          python -S -m tools.check_frozen_requirements sentry
```

Comment:
Just realizing this now, but I think it would be worth to still run this on master?

Same applies to https://github.com/getsentry/getsentry/pull/7898, or we could just do it there.

Reply:
it will error if it needs to make changes so it's essentially the same -- I'm going to delete the `check_frozen_requirements` script after this PR :D